pkg/types/aws/customendpoint.go

```diff
@@ -0,0 +1,11 @@
+// Package aws contains AWS-specific structures for installer
+// configuration and management.
+package aws
+
+// CustomEndpoint store the configuration of a custom url to 
+// override existing defaults of AWS Services.
+// Currently Supports - EC2, IAM, ELB, S3 and Route53. 
+type CustomEndpoint struct{
+	Service       string `json:"service"`
```

Comment:
would like to see field level comments

Reply:
alternatively, I think you could consider not creating these types at all and just storing the endpoints as key value pairs in a map. I think this struct looks really nice and clean, but ultimately terraform wants the map. @abhinavdahiya what do you think? 

Also I think we might be missing some files. I was trying to explore this idea further but I don't see installconfig.go so I couldn't see how this struct is created?

Reply:
My suggestion above would be a significant change... I think in order for the yaml to be unmarshalled to a map it would require the format of the install config to change (not sure if it is possible, but I assume it is).

Reply:
Added additional field level comments.